src/components/visualisation/GoXLRVisualiser.vue

```diff
@@ -384,37 +357,39 @@ export default {
   },
   computed: {
     // variables required for watch
-    isTopLeftSampleRecording() { if (isDeviceMini()) {return false;} return this.isSampleRecording("TopLeft"); },
-    isTopRightSampleRecording() { if (isDeviceMini()) {return false;}return this.isSampleRecording("TopRight"); },
-    isBottomLeftSampleRecording() { if (isDeviceMini()) {return false;}return this.isSampleRecording("BottomLeft"); },
-    isBottomRightSampleRecording() { if (isDeviceMini()) {return false;} return this.isSampleRecording("BottomRight"); },
-    isClearActive() { if (isDeviceMini()) {return false;} return store.getActiveDevice().sampler.clear_active; },
+    isTopLeftSampleRecording() { return !isDeviceMini() && isSampleRecording("TopLeft");},
```

Comment:
Not entirely sure why I didn't do this the first time around, thanks for fixing!